--- libgssapi-0.3/configure.in.orig	2005-05-02 20:18:16.000000000 +0200
+++ libgssapi-0.3/configure.in	2005-05-04 19:19:37.587646056 +0200
@@ -72,8 +72,8 @@
     AC_DEFINE_UNQUOTED(KRB5_VERSION, $K5VERS,
 		       [Define this as the Kerberos version number])
     if test -f $dir/include/gssapi/gssapi_krb5.h -a \
-            \( -f $dir/lib/libgssapi_krb5.a -o \
-            -f $dir/lib/libgssapi_krb5.so \) ; then
+            \( -f $dir/lib/libgssapi_krb5.so -o \
+            -f $dir/lib64/libgssapi_krb5.so \) ; then
        AC_DEFINE(HAVE_KRB5, [], [Define this if you have MIT Kerberos libraries])
        KRBDIR="$dir"
        if test $K5VERS -le 131; then
@@ -86,7 +86,7 @@
     dnl of Heimdal Kerberos on SuSe
     elif test \( -f $dir/include/heim_err.h -o\
                  -f $dir/include/heimdal/heim_err.h \) -a \
-              -f $dir/lib/libroken.a; then
+              \( -f $dir/lib/libroken.so -o -f $dir/lib64/libroken.so \) ; then
        AC_DEFINE(HAVE_HEIMDAL, [], [Define this if you have Heimdal Kerberos libraries])
        KRBDIR="$dir"
        gssapi_lib=gssapi
